pinot-plugins/pinot-file-system/pinot-gcs/pom.xml

```diff
@@ -82,7 +82,6 @@
     <dependency>
       <groupId>com.google.api.grpc</groupId>
       <artifactId>proto-google-common-protos</artifactId>
-      <version>1.18.1</version>
```

Comment:
That seems strange. I removed that line without actually reading it because the actual error didn't point to an issue there but in:

```
[ERROR] Dependency convergence error for com.google.api.grpc:proto-google-common-protos:jar:2.9.0 paths to dependency are:
[ERROR] +-org.apache.pinot:pinot-tools:jar:1.1.0-SNAPSHOT
[ERROR]   +-org.apache.pinot:pinot-common:jar:1.1.0-SNAPSHOT:compile
[ERROR]     +-io.grpc:grpc-protobuf:jar:1.53.0:compile
[ERROR]       +-com.google.api.grpc:proto-google-common-protos:jar:2.9.0:compile
[ERROR] and
[ERROR] +-org.apache.pinot:pinot-tools:jar:1.1.0-SNAPSHOT
[ERROR]   +-org.apache.pinot:pinot-protobuf:jar:1.1.0-SNAPSHOT:compile
[ERROR]     +-io.confluent:kafka-protobuf-serializer:jar:7.4.0:compile
[ERROR]       +-io.confluent:kafka-protobuf-provider:jar:7.4.0:compile
[ERROR]         +-com.google.api.grpc:proto-google-common-protos:jar:2.5.1:compile
[ERROR] and
[ERROR] +-org.apache.pinot:pinot-tools:jar:1.1.0-SNAPSHOT
[ERROR]   +-org.apache.pinot:pinot-protobuf:jar:1.1.0-SNAPSHOT:compile
[ERROR]     +-io.confluent:kafka-protobuf-serializer:jar:7.4.0:compile
[ERROR]       +-io.confluent:kafka-protobuf-types:jar:7.4.0:compile
[ERROR]         +-com.google.api.grpc:proto-google-common-protos:jar:2.5.1:compile
```

So I upgraded the version thinking I only changed minor versions. But now I'm checking again, without removing the version here, and you are right, the error is:

```
[ERROR] Dependency convergence error for com.google.api.grpc:proto-google-common-protos:jar:2.9.6 paths to dependency are:
[ERROR] +-org.apache.pinot:pinot-gcs:jar:1.1.0-SNAPSHOT
[ERROR]   +-com.google.cloud:google-cloud-storage:jar:1.113.1:compile
[ERROR]     +-com.google.api.grpc:proto-google-common-protos:jar:2.9.6:compile
[ERROR] and
[ERROR] +-org.apache.pinot:pinot-gcs:jar:1.1.0-SNAPSHOT
[ERROR]   +-com.google.api.grpc:proto-google-common-protos:jar:1.18.1:compile

```

It seems that [kafka-protobuf-types:5.5.3](https://mvnrepository.com/artifact/io.confluent/kafka-protobuf-types/5.5.3) did not depend on `proto-google-common-protos`, but [kafka-protobuf-types:7.0.0](https://mvnrepository.com/artifact/io.confluent/kafka-protobuf-types/7.0.0) and newer depend on `proto-google-common-protos:2.3` and newer.